src/addon/esmpy/src/esmpy/api/esmpymanager.py

```diff
@@ -121,6 +124,9 @@ def __init__(self, debug=False):
             ESMP_Initialize(logkind=logkind)
             import atexit; atexit.register(self.__del__)
             self.__esmp_initialized = True
+            self.__esmp_keep_mpi = keepMpi
+            if self.__esmp_keep_mpi is None:
+                self.__esmp_keep_mpi = True
```

Comment:
I think `__esmp_keep_mpi` should be set to False in this case (to be consistent with the documentation, and because that seems like the right thing to do if the caller hasn't set this explicitly to False).

But actually, since the keepMpi argument already defaults to False, I wonder if this None handling is really needed. It looks like maybe you followed the example of the `debug` argument in this respect, but I'm having trouble envisioning how this would get set to `None`. So, unless you have a sense that we need this `None` handling, I'd suggest:
- Getting rid of the handling of `None` here
- Deleting the documentation that `None` means False (line 92)
- Doing a quick test of creating an esmpymanager without setting this explicitly; verifying that it ends up as False as expected

Reply:
Ah, yes, I think I set this to true by accident when I had a moment of confusing myself about what `keepMpi` meant. In the case of the `None` handling, yeah, you're correct that I just followed the `debug` argument semantics without thinking too much into it. But looking over it again, I agree that it's not really necessary.

Reply:
Resolved in [b3f27d0](https://github.com/esmf-org/esmf/pull/234/commits/b3f27d0420993c0845dd70567a41e8714c5ac326).

Reply:
Thanks!